src/Modules/Admin/Commands.Admin/GetPowerBIActivityEvents.cs

```diff
@@ -80,11 +83,19 @@ public override void ExecuteCmdlet()
 
             string formattedStartDateTime = $"'{this.StartDateTime}'";
             string formattedEndDateTime = $"'{this.EndDateTime}'";
-            string formattedFilter = this.ActivityType;
-            if (!string.IsNullOrEmpty(this.ActivityType))
+            string formattedFilter = null;
+            if (!string.IsNullOrEmpty(this.ActivityType) && !string.IsNullOrEmpty(this.User))
```

Comment:
Not sure if the possible filters will grow or not, but you could consider adding the strings to a list and then joining them with " and " instead of a long if-else block.